frontend/packages/dev-console/integration-tests/reporter-config.json

```diff
@@ -6,7 +6,7 @@
   },
   "mochawesomeReporterOptions": {
     "reportDir": "../../../gui_test_screenshots/",
-    "reportFilename": "cypress_report_devconsole",
+    "reportFilename": "cypress_report",
```

Comment:
Although I don't think it's required, but to keep consistant with other sub-packages, you can continue to call it `cypress_report_devconsole`.  At some point we may want to categorize/filter the `final cypress-report.html` by sub-project/package and this naming convention might help.

Reply:
Restored the changes